components/usage/pkg/server/server.go

```diff
@@ -65,7 +65,7 @@ func Start(cfg Config) error {
 			return fmt.Errorf("failed to initialize stripe client: %w", err)
 		}
 
-		billingController = controller.NewStripeBillingController(c, pricer)
+		billingController = controller.NewStripeBillingController(c)
```

Comment:
This is the goal of the PR, the billing controller doesn't need the pricer as the WorkspaceInstanceUsage records already have credits used on them.